src/coreclr/jit/gentree.cpp

```diff
@@ -655,6 +657,20 @@ bool GenTree::gtHasReg() const
             }
         }
     }
+    else if (IsMultiRegLclVar())
+    {
+        assert(comp != nullptr);
+        const GenTreeLclVar* lclNode  = AsLclVar();
+        const unsigned       regCount = GetMultiRegCount(comp);
+        for (unsigned i = 0; i < regCount; i++)
```

Comment:
For consistency, add a comment

```suggestion
        // A Multi-reg local vars is said to have regs,
        // if it has valid regs in any of the positions.
        for (unsigned i = 0; i < regCount; i++)
```